factory_metadata.py

```diff
@@ -6,7 +6,7 @@
 from secrets import token_bytes
 from typing import Any
 
-from chia.types.blockchain_format.sized_bytes import bytes32
+from chia_rs.sized_bytes import bytes32
 from chia.util.bech32m import encode_puzzle_hash
```

Comment:
```suggestion
from chia.util.bech32m import encode_puzzle_hash
from chia_rs.sized_bytes import bytes32
```

i _think_ this is what ruff wants